### PR TITLE
feat: close submission in session

### PR DESCRIPTION
**Pull request by aneojgurhem**

### CODE REVIEW

**Review by ngruelaneo — APPROVED** [review]

File: Common/src/Exceptions/SubmissionClosed.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+
+namespace ArmoniK.Core.Common.Exceptions;
+
+[Serializable]
+public class SubmissionClosed : ArmoniKException
```
Comment (lemaitre-aneo) [inline Common/src/Exceptions/SubmissionClosed.cs:23]:
```suggestion
public class SubmissionClosedException : ArmoniKException
```

File: Common/tests/Pollster/AgentTest.cs [hunk 2]
```diff
@@ -678,6 +682,43 @@ await holder.Agent.FinalizeTaskCreation(CancellationToken.None)
...
+                          holder.QueueStorage,
+                          holder.ResultTable,
+                          holder.TaskTable,
+                          sessionData,
+                          holder.TaskData,
+                          holder.Folder,
+                          holder.Token,
+                          NullLogger.Instance);
+
+    Assert.ThrowsAsync<SubmissionClosed>(() => agent.SubmitTasks(new TaskSubmissionRequest[]
```
Comment (lemaitre-aneo) [inline Common/tests/Pollster/AgentTest.cs:709]: `Assert.That` ?

**Review by lemaitre-aneo — COMMENTED** [review]
Is it expected to catch SubmissionClosed and transform it in RpcException only in TaskService, but not in AgentService?

File: Common/tests/Pollster/AgentTest.cs (hunk 2 above)
Comment (aneojgurhem) [reply Common/tests/Pollster/AgentTest.cs:709]: I'll do it with the others!

**Review by lemaitre-aneo — APPROVED** [review]

**Pull request merged**